amazon_eks/README.md

```diff
@@ -16,7 +18,20 @@ Monitoring EKS requires that you set up the Datadog integrations for:
 
 along with integrations for any other AWS services you're running with EKS (e.g., [ELB](https://docs.datadoghq.com/integrations/amazon_elb/))
 
-## Further Reading
+### Log Collection
+
+**Available for Agent >6.0**
+
+The setup is exactly the same as for Kubernetes. 
+To start collecting logs from all your containers, use your Datadog Agent [environment variables](https://docs.datadoghq.com/agent/basic_agent_usage/kubernetes/#log-collection-setup).
 
-Learn more about infrastructure monitoring and all our integrations on [our blog](https://www.datadoghq.com/blog/)
+Take also advantage of DaemonSets to [automatically deploy the Datadog Agent on all your nodes](https://docs.datadoghq.com/agent/basic_agent_usage/kubernetes/#container-installation). 
+
+Follow the [container log collection steps](https://docs.datadoghq.com/logs/log_collection/docker/#option-2-container-installation) to learn more about those environment variables and discover more advanced setup options.
+
+## Troubleshooting
+Need help? Contact [Datadog Support](http://docs.datadoghq.com/help/).
+
+## Further Reading
 
```

Comment:
If theres no further reading, the further reading section should be dropped.  Lets avoid generically linking to datadoghq.com/blog/

That being said in this case there is a relevant blog post:
https://www.datadoghq.com/blog/announcing-eks/

Reply:
@masci @l0k0ms

Reply:
pushing an update

Reply:
https://github.com/DataDog/integrations-core/pull/1722